### ISSUE TITLE
can't search

### ISSUE THREAD

**Issue by sstoner** [brookhong/Surfingkeys#1189]
## Error details

SurfingKeys: 0.9.60

Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.132 Safari/537.36

URL: -

## Context

when I navigate to search something, it happend as follow

chrome-extension://gfbliohnnapiefjpjlpjnehglfpaknnc/pages/error.html

```
This webpage is not available
net::ERR_CONNECTION_CLOSED
https://medium.com/hackernoon/strace-in-60-lines-of-go-b4b76e3ecd64
r to reload.
p to toggle medium.com in list that would be accessed through proxy.
```

**Comment by brookhong** [comment]
Fixed in 0.9.63

**Comment by sstoner** [comment]
> Fixed in 0.9.63

Thank you!
